### PR TITLE
Moarfix

### PR DESCRIPTION
**Pull request by campoy**

### CODE REVIEW

**Comment by campoy** [comment]
Hey @rakyll, could you have a look?

File: 3-dynamic-analysis/2-testing/strings_test.go [hunk 1]
```diff
@@ -67,9 +67,9 @@ func TestIndex_Subtest(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			if idx := strings.Index(tc.text, tc.sub); idx != tc.idx {
 				if tc.idx >= 0 {
-					t.Errorf("%s should contain %s at position %d, not %d", tc.text, tc.sub, tc.idx, idx)
+					t.Fatalf("%s should contain %s at position %d, not %d", tc.text, tc.sub, tc.idx, idx)
 				} else {
```
Comment (rakyll) [inline 3-dynamic-analysis/2-testing/strings_test.go]: You can remove the redundant else now.

File: 3-dynamic-analysis/2-testing/1-go-test.md [hunk 2]
```diff
@@ -187,16 +187,19 @@ TestIndex_Subtest(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			if idx := strings.Index(tc.text, tc.sub); idx != tc.idx {
 				if tc.idx >= 0 {
-					t.Errorf("%s should contain %s at position %d, not %d", tc.text, tc.sub, tc.idx, idx)
+					t.Fatalf("%s should contain %s at position %d, not %d", tc.text, tc.sub, tc.idx, idx)
 				} else {
```
Comment (rakyll) [inline 3-dynamic-analysis/2-testing/1-go-test.md]: You can remove the redundant else now.

Comment (campoy) [inline 3-dynamic-analysis/2-testing/1-go-test.md]: Done

File: 3-dynamic-analysis/2-testing/strings_test.go (hunk 1 above)
Comment (campoy) [inline 3-dynamic-analysis/2-testing/strings_test.go]: Done

**Comment by campoy** [comment]
Can you do a LGTM on the code review section?
Trying to keep this repo as an example of how to manage repos GitHub style 😄 

Thanks!

**Pull request merged**